nixos/modules/config/sysctl.nix

```diff
@@ -59,5 +59,19 @@ in
     # Disable YAMA by default to allow easy debugging.
     boot.kernel.sysctl."kernel.yama.ptrace_scope" = mkDefault 0;
 
+    # Dynamic since kernel v5.12
+    # https://github.com/torvalds/linux/commit/ac7b79fd190b02e7151bc7d2b9da692f537657f3
+    boot.kernel.sysctl."fs.inotify.max_user_instances" =
+      if (config.boot.kernelPackages.kernel.kernelAtLeast "5.12")
+      then mkDefault 2147483647 # INT_MAX, dynamically limited based on available memory
+      else mkDefault 8192; # instead of 128
+
+    # Dynamic since kernel v5.11
+    # https://github.com/torvalds/linux/commit/92890123749bafc317bbfacbe0a62ce08d78efb7a
```

Comment:
This is a 404, so I can't check it.

Reply:
Oh, thank you there was an `a` too much at the end... :/